Makefile

```diff
@@ -45,10 +45,8 @@ ACTIVATE_ENV = source $(shell dirname $(dir $(CONDA)))/bin/activate $(CONDA_ENV)
 
 install: clean ## install dependencies
 	$(ACTIVATE_ENV) && \
-		npm install decktape && \
 		gem update --system && \
-		gem install nokogiri:'1.10.0' -- --use-system-libraries --with-xml=$(CONDA_PREFIX)/lib && \
```

Comment:
Is that line not needed any more?

Reply:
It worked like a charm without this line for me locally. And `$(CONDA_PREFIX)` does not refer to anything in the `Makefile`